packages/visx-scale/src/types/ScaleConfig.ts

```diff
@@ -55,6 +55,13 @@ export type SymlogScaleConfig<Output = DefaultOutput> = CreateScaleConfig<
   'clamp' | 'constant' | 'nice' | 'zero'
 >;
 
+export type RadialScaleConfig<Output = DefaultOutput> = CreateScaleConfig<
+  'radial',
+  ContinuousDomain,
+  Output[],
+  'clamp' | 'nice' | 'round' | 'zero' | 'unknown'
```

Comment:
should `zero` be omitted here since it's not applied in `scales/radial.ts`?

Reply:
Yeah makes sense. Have done the changes

Reply:
+1